### DISCUSSION TITLE
提示不能同时设置宽度

### DISCUSSION THREAD

**Discussion by Vjze** [syf20020816/SurrealismUI#18] [Q&A] [answered]
![image](https://github.com/Surrealism-All/SurrealismUI/assets/99671889/e2278416-7e9b-41a0-9c60-4f9b3a7d0dde)
我检查了一遍我自己是没有设置的最小宽度，我用到的插件也检查了，并没有发现有设置最小宽度。

**Comment by syf20020816** [comment]
可以preview和编译就没有问题

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Vjze ***@***.***>
Sent: Thursday, December 28, 2023 6:19:20 PM
To: Surrealism-All/SurrealismUI ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Surrealism-All/SurrealismUI] 提示不能同时设置宽度 (Issue #17)


image.png (view on web)<https://github.com/Surrealism-All/SurrealismUI/assets/99671889/e2278416-7e9b-41a0-9c60-4f9b3a7d0dde>
我检查了一遍我自己是没有设置的最小宽度，我用到的插件也检查了，并没有发现有设置最小宽度。

―

**Comment by Vjze** [comment]
![image](https://github.com/Surrealism-All/SurrealismUI/assets/99671889/a07bed48-6adf-49ff-afac-8a5f87ff8fac)
![image](https://github.com/Surrealism-All/SurrealismUI/assets/99671889/fb9c35ba-1110-4954-a288-ab4fefde08b9)
预览报错，rust编译也报错

**Comment by syf20020816** [comment, marked as answer]
你使用的是否是SInput，如果是，请在其外部包裹一个Rectangle

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Vjze** [comment]
> 你使用的是否是SInput，如果是，请在其外部包裹一个Rectangle 获取Outlook for Android<https://aka.ms/AAb9ysg>
> […](#)
> ________________________________ From: Vjze ***@***.***> Sent: Thursday, December 28, 2023 6:36:42 PM To: Surrealism-All/SurrealismUI ***@***.***> Cc: ***@***.*** ***@***.***>; Comment ***@***.***> Subject: Re: [Surrealism-All/SurrealismUI] 提示不能同时设置宽度 (Issue #17) image.png (view on web)<https://github.com/Surrealism-All/SurrealismUI/assets/99671889/a07bed48-6adf-49ff-afac-8a5f87ff8fac> image.png (view on web)<https://github.com/Surrealism-All/SurrealismUI/assets/99671889/fb9c35ba-1110-4954-a288-ab4fefde08b9> 预览报错，rust编译也报错 ― Reply to this email directly, view it on GitHub<[#17 (comment)](https://github.com/Surrealism-All/SurrealismUI/issues/17#issuecomment-1871040698)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AV7FXN62I6TUJ26UV3YEHF3YLVDTVAVCNFSM6AAAAABBFKT22WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNZRGA2DANRZHA>. You are receiving this because you commented.Message ID: ***@***.***>

Ok看到了，SInput里面确实有个min-width,解决了，感谢
